app/src/CreateExcelFile/GoogleSearch.js

```diff
@@ -0,0 +1,22 @@
+import { SearchItem } from "./Model";
+
```

Comment:
why is this file inside the directory `CreateExcelFile` ?? 
it has nothing to do with excel 
be smarter about ur file structure
u can make a folder `service` or `utils` and have googlesearch.js and excelutils.js in them

Reply:
okok, wb the Model then? be it a single file in src or also put it in a folder?

Reply:
u can put it in a folder name `model` itself, or leave it as a single file in src as well, upto u..